src/pages/Contact.css

```diff
@@ -0,0 +1,138 @@
+.titleContainer{
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+
+}
+.title{
+    /* style the CONTACT US */
+    width: 468px;
+    height: 68px;
+    /* margin-left: 468px; */
+    margin-left: 242px;
+    top: 159px;
```

Comment:
Usually we don't want to put these kinds of constraints in, as it makes the webpage less responsive; instead, let's just let Flexbox handle it (the earlier declaration in `.titleContainer` of `display: flex ... justify-content: center` will center this automatically)